x/pairing/keeper/msg_server_relay_payment.go

```diff
@@ -235,7 +235,17 @@ func (k msgServer) RelayPayment(goCtx context.Context, msg *types.MsgRelayPaymen
 		// update provider payment storage with complainer's CU
 		err = k.updateProviderPaymentStorageWithComplainerCU(ctx, relay.UnresponsiveProviders, logger, epochStart, relay.SpecId, relay.CuSum, servicersToPair, project.Index)
 		if err != nil {
-			utils.LogLavaEvent(ctx, logger, types.UnresponsiveProviderUnstakeFailedEventName, map[string]string{"err:": err.Error()}, "Error Unresponsive Providers could not unstake")
+			reportedProviders := ""
+			for _, p := range relay.UnresponsiveProviders {
+				reportedProviders += p.String() + ", "
```

Comment:
you can use string.join instead

Reply:
done. see https://github.com/lavanet/lava/commit/6de4aabf932a0ccf06e72626912782c8de252add